cmd/memcached_exporter/main.go

```diff
@@ -82,7 +87,7 @@ func main() {
 	})
 
 	level.Info(logger).Log("msg", "Listening on address", "address", *listenAddress)
-	if err := http.ListenAndServe(*listenAddress, nil); err != nil {
+	if err := https.Listen(srv, *httpsConfig, logger); err != nil {
```

Comment:
As the server is not necessarily serving HTTPS, I'd have chosen a more neutral package name to express this better, e.g. `server`. Following the naming of the golang stdlib and calling the function `ListenAndServe` would have also helped to make it easier for developers to understand what's going on.

Reply:
I am willing to take a PR for renaming Listen to ListenAndServe and add a backwards compatible Listen function.

Reply:
Let's change this here @roidelapluie?

Reply:
I want you fully happy: https://github.com/prometheus/exporter-toolkit/pull/29